example/ledger-keys/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/binance-chain/go-sdk/common/crypto/ledger"
+	"github.com/binance-chain/go-sdk/keys"
+	"github.com/binance-chain/go-sdk/types"
+	"github.com/binance-chain/go-sdk/types/msg"
+	"github.com/binance-chain/go-sdk/types/tx"
+)
+
+func main() {
+	bip44Params := keys.NewFundraiserParams(0, 0)
```

Comment:
Not directly related to this PR, but this should not be called `NewFundraiserParams`. We did not have a fundraiser and Binance Chain bip44 path nodes are different to the Cosmos Hub ones.